ERCS/erc-6538.md

```diff
@@ -2,9 +2,9 @@
 eip: 6538
 title: Stealth Meta-Address Registry
 description: A registry to map addresses to stealth meta-addresses
-author: Matt Solomon (@mds1), Toni Wahrstätter (@nerolation), Ben DiFrancesco (@apbendi), Vitalik Buterin (@vbuterin)
+author: Matt Solomon (@mds1), Toni Wahrstätter (@nerolation), Ben DiFrancesco (@apbendi), Vitalik Buterin (@vbuterin), Gary Ghayrat (@garyghayrat)
 discussions-to: https://ethereum-magicians.org/t/stealth-meta-address-registry/12888
-status: Stagnant
+status: Review
 type: Standards Track
 category: ERC
 created: 2023-01-24
```

Comment:
```suggestion
created: 2023-01-24
requires: 5564
```

Reply:
Added in https://github.com/ethereum/ERCs/pull/259/commits/f7a4aec74220455e12006d99ce1fae7b94655bbc